next/components/forms/segments/EmailVerificationForm/EmailVerificationForm.tsx

```diff
@@ -105,17 +104,26 @@ const EmailVerificationForm = ({ onSubmit, error, onResend, lastEmail, cntDisabl
         variant="category"
         disabled={isSubmitting}
       />
+      {/* turn off timer if error */}
```

Comment:
```suggestion
      {/* don't show timer if error */}
```

as you are not really turning it off, it still runs in background